Sources/EmbraceCore/Capture/OneTimeServices/AppInfoCaptureService.swift

```diff
@@ -61,5 +61,20 @@ class AppInfoCaptureService: ResourceCaptureService {
             key: AppResourceKey.processIdentifier.rawValue,
             value: .string(ProcessIdentifier.current.hex)
         )
+
+        // process start time
+        if let uptime = ProcessMetadata.uptime() {
```

Comment:
This should be `ProcessMetadata.startTime` instead of the `uptime`. We want the absolute time the process started and not the length of time until the AppInfoCaptureService has run.

The `emb-process-launch` span already measures the duration from process launch to the end of the `Embrace.setup` method.

Reply:
good catch